### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by Zeitsperre**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@huard The conda build of xclim v0.12.3 went out a bit too quickly. This build is a dependency fix since xarray base version was still pinned at 0.13.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,40 +10,43 @@ source:
...
-    - pint >=0.8
+    - pint >0.8,<0.10
     - boltons >=19.0
     - scikit-learn >=0.21.3
     - pytest
 
 test:
   imports:
     - xclim
+    - xclim.atmos
```
Comment (huard) [inline recipe/meta.yaml]: xclim.land
xclim.seaIce

Comment (huard) [reply recipe/meta.yaml]: Scrap that, importing xclim now imports atmos, land and seaIce. So you may remove all of them.

Comment (Zeitsperre) [reply recipe/meta.yaml]: Is this true for 0.12? I can revert this in the 0.13 build tomorrow.

Comment (huard) [reply recipe/meta.yaml]: No, you're right.

**Pull request merged**